<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BT trader</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="buttonWidget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>31</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="OrientationButton" name="showQuotesDockButton">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Quotes</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="OrientationButton" name="showDataBrowserDockButton">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Data verkenner</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="OrientationButton" name="showIndicatorsDockButton">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Indicatoren</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="OrientationButton" name="showTransactionsDockButton">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Transacties</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>82</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTime_Scale">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Tijdschaal</string>
    </property>
    <addaction name="action1_minute"/>
    <addaction name="actionHourly"/>
    <addaction name="separator"/>
    <addaction name="actionDaily"/>
    <addaction name="actionWeekly"/>
    <addaction name="actionMonthly"/>
   </widget>
   <widget class="QMenu" name="settingsMenu">
    <property name="title">
     <string>Instellingen</string>
    </property>
    <addaction name="actionBTTraderSettings"/>
   </widget>
   <widget class="QMenu" name="menuBron">
    <property name="title">
     <string>Bron</string>
    </property>
    <addaction name="actionBron_toevoegen"/>
   </widget>
   <widget class="QMenu" name="menuIndicators">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Indicatoren</string>
    </property>
    <addaction name="actionMoving_Average"/>
    <addaction name="action2_Moving_Averages"/>
    <addaction name="action3_Moving_Averages"/>
    <addaction name="actionWinst_volger"/>
    <addaction name="actionBollinger_Bands"/>
    <addaction name="actionPrice_Channel"/>
    <addaction name="actionTrendLine"/>
    <addaction name="actionSupportCeilingLine"/>
    <addaction name="actionMesa_Instantaneous_Trend"/>
    <addaction name="separator"/>
    <addaction name="actionRSI"/>
    <addaction name="actionStochastic"/>
    <addaction name="actionVolatility"/>
    <addaction name="actionER"/>
    <addaction name="actionADX"/>
    <addaction name="actionDominant_Cycle"/>
    <addaction name="actionMesa_Trend_Vigor"/>
    <addaction name="actionMesa_SNR"/>
   </widget>
   <widget class="QMenu" name="strategyMenu">
    <property name="title">
     <string>Strategie</string>
    </property>
    <addaction name="actionOpslaan"/>
    <addaction name="actionImporteren"/>
    <addaction name="actionExporteren"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuBron"/>
   <addaction name="menuTime_Scale"/>
   <addaction name="menuIndicators"/>
   <addaction name="settingsMenu"/>
   <addaction name="strategyMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="DockWidget" name="quotesDockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable</set>
   </property>
   <property name="windowTitle">
    <string>Overzicht</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="VerticalScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>99</width>
          <height>435</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="DockWidget" name="indicatorDockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="windowTitle">
    <string>Indicatoren</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="DockWidget" name="transactionsDockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable</set>
   </property>
   <property name="windowTitle">
    <string>Transacties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="DockWidget" name="dataBrowserDockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable</set>
   </property>
   <property name="windowTitle">
    <string>Data verkenner</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionDot">
   <property name="text">
    <string>Dot</string>
   </property>
  </action>
  <action name="actionSolid">
   <property name="text">
    <string>Solid</string>
   </property>
  </action>
  <action name="actionCandle">
   <property name="text">
    <string>Candle</string>
   </property>
  </action>
  <action name="actionTick_by_tick">
   <property name="text">
    <string>Tick by Tick</string>
   </property>
  </action>
  <action name="action1_minute">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Minuten</string>
   </property>
  </action>
  <action name="actionHourly">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Uren</string>
   </property>
  </action>
  <action name="actionDaily">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dagen</string>
   </property>
  </action>
  <action name="actionMoving_Average">
   <property name="text">
    <string>Moving Average</string>
   </property>
  </action>
  <action name="actionZoom_goto">
   <property name="text">
    <string>zoom goto</string>
   </property>
  </action>
  <action name="actionRSI">
   <property name="text">
    <string>RSI (Relative Strength Index)</string>
   </property>
  </action>
  <action name="actionTrendLine">
   <property name="text">
    <string>Trend Lijn</string>
   </property>
  </action>
  <action name="actionWeekly">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Weken</string>
   </property>
  </action>
  <action name="actionVolatility">
   <property name="text">
    <string>Volatility</string>
   </property>
  </action>
  <action name="actionER">
   <property name="text">
    <string>ER (Efficientie Ratio)</string>
   </property>
  </action>
  <action name="action2_Moving_Averages">
   <property name="text">
    <string>2 Moving Averages</string>
   </property>
  </action>
  <action name="action3_Moving_Averages">
   <property name="text">
    <string>3 Moving Averages</string>
   </property>
  </action>
  <action name="actionBron_toevoegen">
   <property name="text">
    <string>Bron toevoegen</string>
   </property>
  </action>
  <action name="actionWinst_volger">
   <property name="text">
    <string>Koers volger</string>
   </property>
  </action>
  <action name="actionBollinger_Bands">
   <property name="text">
    <string>Bollinger Bands</string>
   </property>
  </action>
  <action name="actionADX">
   <property name="text">
    <string>ADX (Average Directional Movement Index)</string>
   </property>
  </action>
  <action name="actionPrice_Channel">
   <property name="text">
    <string>Price Channel</string>
   </property>
  </action>
  <action name="actionDominant_Cycle">
   <property name="text">
    <string>Mesa Dominant Cycle</string>
   </property>
  </action>
  <action name="actionMesa_Instantaneous_Trend">
   <property name="text">
    <string>Mesa Instantaneous Trend</string>
   </property>
  </action>
  <action name="actionMesa_Trend_Vigor">
   <property name="text">
    <string>Mesa Trend Vigor</string>
   </property>
  </action>
  <action name="actionMesa_SNR">
   <property name="text">
    <string>Mesa SNR</string>
   </property>
  </action>
  <action name="actionStochastic">
   <property name="text">
    <string>Stochastic</string>
   </property>
  </action>
  <action name="actionOpslaan">
   <property name="text">
    <string>Opslaan</string>
   </property>
  </action>
  <action name="actionImporteren">
   <property name="text">
    <string>Importeren</string>
   </property>
  </action>
  <action name="actionExporteren">
   <property name="text">
    <string>Exporteren</string>
   </property>
  </action>
  <action name="actionMonthly">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Maanden</string>
   </property>
  </action>
  <action name="actionSupportCeilingLine">
   <property name="text">
    <string>Steun/Weerstand Lijn</string>
   </property>
  </action>
  <action name="actionBTTraderSettings">
   <property name="text">
    <string>BT Trader instellen</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VerticalScrollArea</class>
   <extends>QScrollArea</extends>
   <header location="global">verticalscrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global">dockwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrientationButton</class>
   <extends>QPushButton</extends>
   <header location="global">orientationbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
